twilio.js

```diff
@@ -2,7 +2,7 @@ var twilio = Npm.require('twilio');
 
 var lookups = new Mongo.Collection('twilio.lookups');
 
-Twilio = function(options) {
+Twilio = function (options) {
```

Comment:
That's the "official" Meteor jsbeautify rules, no? I don't have a preference either way, my editor just auto formats it according to the `.jsbeautifyrc` file